post-til-private/src/main/java/no/difi/meldingsutveksling/ptp/EmptyKvittering.java

```diff
@@ -0,0 +1,37 @@
+package no.difi.meldingsutveksling.ptp;
+
+import net.logstash.logback.marker.LogstashMarker;
+import net.logstash.logback.marker.Markers;
+import no.difi.meldingsutveksling.ServiceIdentifier;
+import no.difi.meldingsutveksling.receipt.ExternalReceipt;
+import no.difi.meldingsutveksling.receipt.MessageReceipt;
+
+public class EmptyKvittering implements ExternalReceipt {
+
+    public static final String EMPTY = "empty";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "empty" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1192)